src/views/Viewer.vue

```diff
@@ -170,6 +178,9 @@ export default {
 	}),
 
 	computed: {
+		canDelete() {
+			return this.currentFile.permissions.includes('D')
```

Comment:
Maybe check if permissions is here as well?
```suggestion
			return 'permissions' in this.currentFile
				&& this.currentFile.permissions.includes('D')
```